### ISSUE TITLE
[Accessibility]: Incorrectly clickable elements in upload timeline

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#44553]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

On the upload information page, the steps in the timeline have clickable elements but clicking them doesn't do anything.

### Expected behavior

Either the elements should not be clickable, or clicking them should expand a section with more information

### Additional details

https://user-images.githubusercontent.com/93103176/202482908-6bd9c8b7-abd8-4545-886e-3d5a9ee444d0.mov

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
